src/main/java/org/gitlab4j/api/models/Iteration.java

```diff
@@ -0,0 +1,116 @@
+package org.gitlab4j.api.models;
+
+import java.util.Date;
+
+import org.gitlab4j.api.utils.JacksonJsonEnumHelper;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class Iteration {
+	public enum IterationState {
+		Opened, Upcomming, Current, Closed;
+
+		private static JacksonJsonEnumHelper<IterationState> enumHelper = new JacksonJsonEnumHelper<>(IterationState.class, true, true);
+
+        @JsonCreator
+        public static IterationState forValue(String value) {
+            return enumHelper.forValue(value);
+        }
+
+        @JsonValue
+        public String toValue() {
+            return (enumHelper.toString(this));
+        }
+
+        @Override
+        public String toString() {
+            return (enumHelper.toString(this));
+        }
+	}
+
+	private Long id;
+	private Long iid;
+	private Long sequence;
+	private Long groupId;
+	private String title;
+	private String descripion;
+	private IterationState state;
```

Comment:
In the json, the state is an integer:

https://gitlab.com/gitlab-org/gitlab/-/blob/4db027c5a18a8638bb0e6103b49010975058a9f8/doc/api/iterations.md#L56

Reply:
From the code:
https://gitlab.com/gitlab-org/gitlab/-/blob/4db027c5a18a8638bb0e6103b49010975058a9f8/ee/app/models/iteration.rb#L14-18

It seems that there are only 3 possible values.